packages/inscription-view/src/components/parts/request/RequestPart.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('RequestPart', () => {
     renderPart();
     await CollapsableUtil.assertClosed('Permission');
     await CollapsableUtil.toggle('Permission');
-    expect(screen.queryByLabelText('Role')).not.toBeInTheDocument();
+    expect(screen.queryByLabelText('Role')).toBeInTheDocument();
```

Comment:
as the eslint rule says: if the element expected to be present you should use getBy instead of queryBy. only if not present queryBy is needed. 
I think this is because of performance reasons...
if you have the ESLint plugin installed in vscode this should also be shown directly and maybe autofixed on save.